src/KeyVault/KeyVault/ChangeLog.md

```diff
@@ -18,6 +18,8 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Removed two aliases: `New-AzKeyVaultCertificateAdministratorDetails` and `New-AzKeyVaultCertificateOrganizationDetails`
```

Comment:
Caution: the first two line of `Upcoming Release` target S169, the rest target S168